lms/djangoapps/teams/tests/test_views.py

```diff
@@ -597,10 +597,18 @@ def test_without_course_id(self):
     @ddt.data(
         (None, 200, ['Coal Power', 'Nuclear Power', u'sólar power', 'Wind Power'], 'name'),
         ('name', 200, ['Coal Power', 'Nuclear Power', u'sólar power', 'Wind Power'], 'name'),
+        ('team_count', 200, ['Nuclear Power', u'sólar power', 'Coal Power', 'Wind Power'], 'team_count'),
```

Comment:
Note to reviewers: "Coal Power" and "Wind Power" have the same team_count, so this is also verifying the secondary sort by name.